calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java

```diff
@@ -579,5 +586,27 @@ static String getLocalPMString(DateTime now) {
 		DateTimeFormatter df = new DateTimeFormatterBuilder().appendHalfdayOfDayText().toFormatter().withLocale(locale);
 		return df.print(dt);
 	}
-	
+
+	/**
+	 * get Map for eventType and image
+	 * @param configProps
+	 * @return
+	 */
+	public Map<String, String> getEventImageMap(Properties configProps){
+		if(eventIconMap.size() == 0){
+			//load keys for locale root
+			Map<String, String> eventKeyMap = new HashMap<String, String>();
+			ResourceBundle bundle = ResourceBundle.getBundle("calendar", Locale.ROOT);
```

Comment:
In 'chef_calendar.vm' alt for the image is read from the user's locale so it displays correct alt message.